support/dynamics/gp/year-end-closing-procedures-gl.md

```diff
@@ -2,7 +2,7 @@
 title: Year-end closing procedures for GL
 description: Describes the recommended year-end closing procedures for General Ledger in Microsoft Dynamics GP.
 ms.reviewer: dbader, LMuelle
```

Comment:
Hi @theley502, for this article, shall we also update the ms.reviewer to your MS alias?